src/Application/Adapters/Drupal/Preset.php

```diff
@@ -19,7 +20,7 @@ public static function getName(): string
         return 'drupal';
     }
 
-    public static function get(): array
+    public static function get(?Composer $composer): array
```

Comment:
Is this syntax support in php 7.2 ?

Reply:
It was part of 7.1 ([ref](https://www.php.net/releases/7_1_0.php)).

However our phpstan style does not like it, should we allow it or change it to not use that syntax?

Reply:
Can you explain me the context why we have `?Composer $composer`?

Reply:
If we are not able to find a composer file, then Composer will be null. However I think I'll go ahead and change it to an Composer object with no data.